target/linux/mpc85xx/files/arch/powerpc/boot/dts/firebox-t1x.dtsi

```diff
@@ -221,6 +233,26 @@
 	};
 
 	pci1: pcie@ffe0a000 {
-		status = "disabled";
+		reg = <0 0xffe0a000 0 0x1000>;
+		ranges = <0x2000000 0x0 0x80000000 0 0x80000000 0x0 0x20000000
+			  0x1000000 0x0 0x00000000 0 0xffc00000 0x0 0x10000>;
+		pcie@0 {
+			ranges = <0x2000000 0x0 0x80000000
+				  0x2000000 0x0 0x80000000
+				  0x0 0x20000000
+
+				  0x1000000 0x0 0x0
+				  0x1000000 0x0 0x0
+				  0x0 0x100000>;
+
+			ath9k: wifi@0,0 {
+				compatible = "pci168c,0033";
+				reg = <0x0000 0 0 0 0>;
+				#gpio-cells = <2>;
```

Comment:
I don't see the gpio-controller being used anywhere.

Reply:
Correct, it's unused. But if I remove it from the DTS (I actually had it removed in earlier pushes of this PR), then the ath9k driver will expose a nonexisting "ath9k-led".

Reply:
Any additional comments? Should I set to "Resolved"?

Reply:
Sure